src/content/en/fundamentals/instant-and-offline/get-started/graphicalcontent-4.md

```diff
@@ -0,0 +1,186 @@
+﻿project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-10-18 #}
+{# wf_published_on: 2017-10-18 #}
+
+# Graphical Content {: .page-title }
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+
+Web users are a visual bunch, and we rely on images to support web content. Like text, graphical content is a critical component in conveying information in a web page or app. All kinds of images, from charts and graphs to icons and arrows to mugshots and maps, provide instantaneous data and improve reader comprehension and retention. But unlike text, images require considerable time and bandwidth to download and render. Graphical content can easily account for 60%-85% of a typical website's total bandwidth (source: it depends on who you ask).
+
+Clearly, one of the major ways to get a significant performance boost is to reduce the amount of time images take to load. Let's examine some ways to tackle this problem.
+
+## Necessity
+
+The first question to ask is whether you really need an image, and it's a question that apparently isn't asked enough. Far too many sites of all kinds put large, and mostly useless, images "above the fold". This slows down the page load and makes readers scroll halfway down the page before they even get to meaningful content. That's bad, bad design.
```

Comment:
Because this is coming from a Google "voice" I wonder if there's value in putting the ball more in their park vs passing judgement. e.g rather than saying that it's bad design to include large images above the fold:

1. Suggest they ask whether the image is critical (you've done this)
2. Evaluate whether the image needs to be loaded right away (can it be lazy loaded?)
3. If the image needs to be loaded right away, encourage them to compress it accordingly so it limits how much it slows down their pages.
4. Remind them that every additional request that could slow down page load should offer value - if it can't, consider dropping it. 

I otherwise really like the paragraph right below this!